.github/workflows/pr.yml

```
@@ -9,7 +9,7 @@ concurrency:
 
 jobs:
   docs:
-    uses: "./.github/workflows/docs.yml"
+    uses: ./.github/workflows/docs-ci.yml
```

Please keep proper quoting as a good habit.
```suggestion
    uses: "./.github/workflows/docs-ci.yml"
```